tests/testament/tester.nim

```diff
@@ -474,7 +474,7 @@ proc main() =
   case action
```

Comment:
Why does this file exist and wasn't moved over to ``testament/tester.nim``?

Reply:
oops, not sure how that happened. fixed.